swc/private/swc.bzl

```diff
@@ -110,7 +110,7 @@ def _replace_ext(f, ext_map):
         return new_ext
```

Comment:
WDYT about the TODO above, loading `ts_lib.bzl` instead of having diverging implementations?

Reply:
Maybe for some things where we want to ensure the logic aligns? I worry about adding that dependency though...